apps/editing-toolkit/editing-toolkit-plugin/wpcom-universal-themes/index.php

```diff
@@ -114,6 +114,9 @@ function unload_core_fse() {
 	}
 	add_filter( 'block_editor_settings_all', __NAMESPACE__ . '\hide_fse_blocks' );
 	add_filter( 'block_editor_settings_all', __NAMESPACE__ . '\hide_template_editing', 11 );
+
+	// always hide the WP core Site Editor.
+	add_action( 'admin_menu', __NAMESPACE__ . '\hide_core_site_editor' );
```

Comment:
actually, also now wondering if we should remove this action in the `load_core_fse` function.

Reply:
Good point! Updating the fse theme detection logic in https://github.com/Automattic/wp-calypso/pull/60499/commits/1068d46d657defedb9cec7edf292c4925e28258d got this working as expected.